src/make_data/banner_intervention/email_selects_results_page.sql

```diff
@@ -0,0 +1,43 @@
+/* This query extracts the total number and proportion of sessions that select
+   the checker via email that also access the results page
+*/
+
+
+DECLARE start_date STRING DEFAULT "20210803";
+DECLARE end_date STRING DEFAULT "20210816";
+
+CREATE OR REPLACE TABLE `govuk-bigquery-analytics.banner_intervention.email_results_page` AS
+
+-- All distinct sessions that access the checker via the email
+
+WITH
+  sessions_select_email AS (
+    SELECT DISTINCT
+      CONCAT(fullVisitorId, "-", visitId) AS sessionId
+    FROM `govuk-bigquery-analytics.87773428.ga_sessions_*`
+    CROSS JOIN UNNEST(hits) AS hits
+    WHERE _TABLE_SUFFIX BETWEEN start_date AND end_date
+    AND STARTS_WITH(hits.page.pagePath, "/next-steps-for-your-business?crn")
+  ),
+
+-- All distinct sessions that access the checker via the email that also get to the
+-- end of the checker (i.e. the results page)
+
+  sessions_results_page AS (
+    SELECT DISTINCT
+      CONCAT(fullVisitorId, "-", visitId) AS sessionId
+    FROM `govuk-bigquery-analytics.87773428.ga_sessions_*`
+    CROSS JOIN UNNEST(hits) AS hits
+    WHERE _TABLE_SUFFIX BETWEEN start_date AND end_date
+      AND STARTS_WITH(hits.page.pagePath, "/next-steps-for-your-business/results")
+      AND CONCAT(fullVisitorId, "-", visitId) IN (SELECT sessionId FROM sessions_select_email)
+  )
+
+-- The proportion of sessions that select the checker via the email that also get
+-- to the end of the checker (i.e. the results page)
+
+SELECT
+  COUNT(DISTINCT sessionId) AS allSessionsThatSelectChecker,
```

Comment:
same comment about `SELECT DISTINCT` and `COUNT DISTINCT` here and src/make_data/banner_intervention/banner_selects_results_page.sql , usually I'd prefer `COUNT DISTINCT` over `SELECT DISTINCT`, but I get why you use it for `sessions_select_email` so you can look up `sessionId`s more easily later. 

Perhaps drop `DISTINCT` from `SELECT` where we don't use a table as a lookup, and by default do the distinct-ing when counting? This way if someone skips to the end of the query they can be reassured we are deduplicating sessionIds.

Reply:
Many thanks @suganya-s - that's a helpful tip re: by default do the distnct-ing when counting (and when not using a table as a lookup). That makes sense, as that's when we want distinct sessions counted. 

I've changed `SELECT DISTINCT` to `SELECT` for the table `sessions_results_page` in both scripts

Please see commit https://github.com/alphagov/govuk-entity-personalisation/pull/36/commits/2edffcdfd46fc3011345d24f79e421dc2640de82